reference-architectures/deploy-arch-ibm-slz-ocp.md

```diff
@@ -0,0 +1,124 @@
+    url: "https://cloud.ibm.com/docs/deployable-reference-architectures?topic=deployable-reference-architectures-vsi-ra"
+    description: "A deployable architecture that is based on the IBM Cloud for Financial Services reference and that provides virtual servers in a secure VPC for your workloads."
+
+use-case: Cybersecurity
+industry: Banking,FinancialSector
+compliance: FedRAMP
+
+content-type: reference-architecture
+
+---
+
+{{site.data.keyword.attribute-definition-list}}
+
+<!--
+Don't include "reference architecture" in the following title.
+Specify a title based on a use case. If the architecture has a module
+or tile in the IBM Cloud catalog, match the title to the catalog. See
+https://test.cloud.ibm.com/docs/solution-as-code?topic=solution-as-code-naming-guidance.
+-->
+
+# OpenShift Container Platform on VPC
+{: #ocp-ra}
+{: toc-content-type="reference-architecture"}
+{: toc-industry="Banking,FinancialSector"}
+{: toc-use-case="Cybersecurity"}
+{: toc-compliance="FedRAMP"}
+
+OpenShift Container Platform on VPC for regulated industries is a deployable architecture solution that is based on the IBM Cloud for Financial Services reference architecture. It creates secure and compliant Red Hat OpenShift Container Platform workload clusters on a Virtual Private Cloud (VPC) network.
+
+## Architecture diagram
+{: #ra-ocp-architecture-diagram}
+
+![Architecture diagram of the OpenShift Container Platform on VPC deployable architecture.](roks.drawio.svg "Architecture diagram of OpenShift Container Platform on VPC for regulated industries deployable architecture"){: caption="Figure 1. Single region architecture diagram for Red Hat OpenShift Container Platform on VPC on IBM Cloud" caption-side="bottom"}
+
+## Design requirements
+{: #ra-ocp-design-requirements}
+
+![Design requirements for Secure infrastructure on VPC for regulated industries](heat-map-deploy-arch-slz-ocp.svg "Design requirements"){: caption="Figure 2. Scope of the design requirements" caption-side="bottom"}
+
+<!--
+TODO: Add the typical use case for the architecture.
+The use case might include the motivation for the architecture composition,
+business challenge, or target cloud environments.
+-->
+
+## Components
+{: #ra-ocp-components}
+
+### VPC architecture decisions
+{: #ra-ocp-components-arch}
+
+| Requirement | Component | Reasons for choice | Alternative choice |
+|-------------|-----------|--------------------|--------------------|
+| * Ensure public internet connectivity  \n * Isolate most OpenShift clusters so that they are not reachable directly from the public internet | Edge VPC service|Create a separate VPC service where public internet connectivity is allowed to be configured | |
+| Provide compute, storage, and network services to enable the application application provider's administrators to monitor, operate, and maintain the environment | Management VPC service | Create a separate VPC service where SSH connectivity from outside is allowed | |
```

Comment:
For Management VPC service, I would state it is for access and management/tooling for your workload that is deployed in your Workload VPC

Reply:
OK. Updated. I had pulled in the language from  https://cloud.ibm.com/docs/framework-financial-services?topic=framework-financial-services-vpc-architecture-about#vpc-arch-diagram